www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Folder Structure
+description: Folder structure of a newly scaffolded Create T3 App
+layout: ../../layouts/docs.astro
+---
+
+The following is the folder structure of a newly scaffolded Create T3 App, with all options selected.
+
+Further down, the description of each folder indicates its purpose and if it is only included with selected libraries.
+
+```
+.
+├─ prisma
+│  └─ schema.prisma
+├─ public
+│  └─ favicon.ico
+└─ src
+   ├─ env
+   │  ├─ client.mjs
+   │  ├─ schema.mjs
+   │  └─ server.mjs
+   ├─ pages
+   │  ├─ _app.tsx
+   │  └─ index.tsx
+   ├─ styles
+   │  └─ globals.css
+   ├─ types
+   │  └─ next-auth.d.ts
+   ├─ utils
+   │  └─ trpc.ts
+   ├─ pages
+   │  └─ api
+   │     ├─ examples.ts
+   │     └─ restricted.ts
+   ├─ server
+   │  ├─ common
+   │  │  └─ get-server-auth-session.ts
+   │  ├─ db
+   │  │  └─ client.ts
+   │  └─ trpc
+   │     ├─ context.ts
+   │     └─ trpc.ts
+   ├─ pages
+   │  └─ api
+   │     ├─ auth
+   │     │  └─ [...nextauth].ts
+   │     └─ trpc
+   │        └─ [trpc].ts
+   └─ server
+      └─ trpc
+         └─ router
+            ├─ _app.ts
+            ├─ auth.ts
+            └─ example.ts
+├─ .env
+├─ .env-example
+├─ .eslintrc.json
+├─ .gitignore
+├─ README.md
+├─ next-env.d.ts
+├─ next.config.mjs
+├─ package.json
+├─ postcss.config.cjs
+├─ prettier.config.cjs
+├─ tailwind.config.cjs
+├─ tsconfig.json
+```
+
+## `prisma`
+
```

Comment:
Can also add that other prisma related files like migrations and seed scripts go here

Reply:
Thanks heaps for the review, have added a bit more on migrations/seed scripts to the `prisma` folder section, as well as updated all of the relative URLs to exclude /en :)

Reply:
`prisma` folder info fixed in https://github.com/t3-oss/create-t3-app/pull/731/commits/d66b31e9a0dcfa112cd6e15d9038bb0bf40a1010